### DISCUSSION TITLE
Telco Community Gathering - KubeCon EU 2023

### DISCUSSION THREAD

**Discussion by taylor** [lfn-cnti/bestpractices#257] [Events]
This is an open discussion area for the cloud native telco community gathering at KubeCon EU in Amsterdam on Tuesday, April 18th, 2023. This will be in the afternoon following Cloud Native Telco Day

See the related github issue https://github.com/cncf/cnf-wg/issues/256

If you have topics you would like to discuss or agenda for our working session (such as a term to contribute to the CNCF glossary) they can be posted here for open discussion and/or to the github issue as a topic to add to the agenda.

Please fill out this form if interested in the event https://forms.gle/bam6VEk22B5Q5Vy7A

<center>
<img alt="visual representation of the telco community gathering with tulips in Amsterdam with people sitting around together" src="https://user-images.githubusercontent.com/26697/225457326-9b63717c-023f-4dec-966d-a12793188df0.png" width="200" height="200"/>
</center>

**Comment by tomkivlin** [comment]
Someone suggested I bring along a topic of "Probing CNFs in containerised environments" - can do it as a quick talk or a discussion point - will come prepared for both.
